bgpd/bgp_vty.c

```diff
@@ -6586,47 +6513,73 @@ DEFUN (vpn_policy_route_map,
 	return CMD_SUCCESS;
 }
 
-DEFUN (vpn_policy_no_route_map,
-       vpn_policy_no_route_map_cmd,
-       "no route-map <fromvpn|tovpn>",
+ALIAS (af_route_map_vpn_imexport,
+       af_no_route_map_vpn_imexport_cmd,
+       "no route-map vpn <import|export>$direction_str",
        NO_STR
        "Specify route map\n"
-       "fromvpn: core vpn -> this vrf\n"
-       "tovpn: this vrf -> core vpn\n")
+       "Between current address-family and vpn\n"
+       "For routes leaked from vpn to current address-family\n"
+       "For routes leaked from current address-family to vpn\n")
+
+/* This command is valid only in a bgp vrf instance or the default instance */
+DEFPY (bgp_imexport_vpn,
+       bgp_imexport_vpn_cmd,
+       "[no] <import|export>$direction_str vpn",
+       "Export routes to another routing protocol\n"
+       "to VPN RIB per vpn-policy")
```

Comment:
Please add the correct help strings,  we are missing some here.

Reply:
Thanks for catching; fixed.